tex/pgfcirctripoles.tex

```diff
@@ -1644,15 +1644,13 @@
         \pgfpathlineto{\pgfpoint{\pgf@circ@res@left-\pgfhorizontaltransformationadjustment*.5*\pgflinewidth}{\pgf@circ@res@up+\pgf@circ@res@down}}
         \pgfusepath{draw}
     \fi
-        \ifpgf@circuit@pmos@nocircle\else
```

Comment:
This is the hunk I was referring to... I think it's wrong, it removes a conditional added in PR #240

Reply:
Hmm, i will check.

Reply:
Hmm, I missed this one, created a new fork, so will continue. Shall I withdraw this pull request?